Source/SwiftLintFramework/Rules/ImplicitGetterRule.swift

```diff
@@ -0,0 +1,135 @@
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "implicit_getter",
+        name: "Implicit Getter",
+        description: "Computed read-only properties should avoid using the get keyword.",
+        nonTriggeringExamples: [
+            classScoped("var foo: Int {\n get {\n return 3\n}\n set {\n _abc = newValue \n}\n}"),
+            classScoped("var foo: Int {\n return 20 \n} \n}"),
+            classScoped("static var foo: Int {\n return 20 \n} \n}"),
+            classScoped("static foo: Int {\n get {\n return 3\n}\n set {\n _abc = newValue \n}\n}"),
+            classScoped("var foo: Int"),
+            classScoped("var foo: Int {\n return getValueFromDisk() \n} \n}"),
+            classScoped("var foo: String {\n return \"get\" \n} \n}"),
+            "protocol Foo {\n var foo: Int { get }\n",
+            "protocol Foo {\n var foo: Int { get set }\n",
+            "class Foo {\n" +
+            "  var foo: Int {\n" +
+            "    struct Bar {\n" +
+            "      var bar: Int {\n" +
+            "        get { return 1 }\n" +
+            "        set { _ = newValue }\n" +
+            "      }\n" +
+            "    }\n" +
+            "    return Bar().bar\n" +
+            "  }\n" +
+            "}\n"
+        ],
+        triggeringExamples: [
+            classScoped("var foo: Int {\n ↓get {\n return 20 \n} \n} \n}"),
+            classScoped("static var foo: Int {\n ↓get {\n return 20 \n} \n} \n}")
+        ]
+    )
+
+    public func validateFile(file: File) -> [StyleViolation] {
+        let getTokens = file.syntaxMap.tokens.filter { token -> Bool in
+            guard SyntaxKind(rawValue: token.type) == .Keyword else {
+                return false
+            }
+
+            guard let tokenValue = file.contents.substringWithByteRange(start: token.offset,
+                length: token.length) else {
+                    return false
+            }
+
+            return tokenValue == "get"
+        }
+
+        let violatingTokens = getTokens.filter { token -> Bool in
+            // the last element is the deepest structure
+            guard let dictionary =
+                variableDeclarationsFor(token.offset, structure: file.structure).last else {
+                    return false
+            }
+
+            // If there's a setter, `get` is allowed
+            return dictionary["key.setter_accessibility"] == nil
+        }
+
+        return violatingTokens.map { token in
+            // Violation found!
+            let location = Location(file: file, byteOffset: token.offset)
+
+            return StyleViolation(ruleDescription: self.dynamicType.description,
+                severity: configuration.severity,
+                location: location
+            )
+        }
+    }
+
+    private func variableDeclarationsFor(byteOffset: Int, structure: Structure) ->
+                                                             [[String : SourceKitRepresentable]] {
```

Comment:
I'm not a fan of breaking line here, but I couldn't think of other ways to respect the 100 line limit 😬